snowpack/src/build/import-css.ts

```diff
@@ -37,3 +37,10 @@ export async function cssModules({
 export function cssModuleJSON(url: string) {
   return cssModuleNames.get(url) || '{}';
 }
+
+/** Should this file get CSS Modules? */
+export function needsCSSModules(url: string) {
+  return (
+    url.endsWith('.module.css') || url.endsWith('.module.scss') || url.endsWith('.module.sass')
+  );
+}
```

Comment:
Just double checking, these are the only suffixes we need? Do we support user-provided preprocessors (`.module.less`, etc)?

Reply:
You may be right—this may change behavior for `.module.less`, but that support probably has other untested problems. `.module.*` in general is a Snowpack internal behavior we never really exposed to external plugins, so there’s never really been a user-provided interface for it, no.